source/styleguide/brand.liquid

```diff
@@ -0,0 +1,52 @@
+---
+title: Brand Guidelines
+---
+
+{% markdown %}
```

Comment:
@sanajaved7 FWIW, I'm starting to default to using `.liquid` unless a page is really all or almost all Markdown.